index.js

```diff
@@ -116,30 +126,44 @@ function applyAttributes(t, openingElement, componentName, sourceFileName, compo
   }
   if (!openingElement.node.attributes) openingElement.node.attributes = {}
 
+  const elementName = openingElement.node.name.name || 'unknown'
+  const ignoredComponentFromOptions = ignoreComponentsFromOption && ignoreComponentsFromOption.find(component => {
+    let match = true;
+    if (component[0] !== '*' && component[0] !== sourceFileName) match = false;
+    else if (component[1] !== '*' && component[1] !== componentName) match = false;
+    else if (component[2] !== '*' && component[2] !== elementName) match = false;
+    
+    return match
+  }) !== undefined
```

Comment:
This feels a little confusing.. but I'm not sure I have a good suggestion for making it better. I believe the logic is correct, after thinking about it for a bit.

Reply:
Maybe do `!!` instead of comparing to undefined?